assets/airports/ksav.json

```diff
@@ -0,0 +1,2124 @@
+        "BASLE" : ["N32d09m43.362", "W80d45m48.490"],
+        "BROSE" : ["N32d17m12.687", "W81d40m56.549"],
+        "CATHY" : ["N31d40m11.624", "W81d10m17.264"],
+        "CESOR":  ["N32d36m13.089", "W81d38m32.314"],
+        "CHS"  :  ["N32d53m39.543", "W80d02m16.121"],
+        "CINBA":  ["N32d17m06.491", "W80d58m40.208"],
+        "ESENT":  ["N32d09m28.130", "W80d55m03.779"],
+        "FAPLI":  ["N32d15m12.715", "W81d20m58.781"],
+        "FILOX":  ["N32d19m42.713", "W81d38m27.873"],
+        "FRIES":  ["N32d13m52.696", "W80d47m37.956"],
+        "GIPPL":  ["N31d22m53.957", "W81d09m53.696"],
+        "GULLA":  ["N32d56m17.705", "W80d29m57.116"],
+        "HARPS":  ["N31d41m29.560", "W81d15m15.735"],
+        "HENIG":  ["N32d10m17.993", "W80d48m13.464"],
+        "JAMEB":  ["N32d22m10.265", "W81d58m36.549"],
+        "JANIE":  ["N31d51m33.930", "W81d17m44.602"],
+        "JROSS":  ["N32d42m40.008", "W80d37m37.990"],
+        "KUCES":  ["N32d42m40.008", "W80d37m37.990"],
+        "LOTTS":  ["N32d20m11.534", "W81d51m17.258"],
+        "LUCNE":  ["N32d49m46.236", "W81d12m24.162"],
+        "MAYAR":  ["N32d14m38.620", "W81d32m03.949"],
+        "MIZLU":  ["N32d01m04.134", "W81d11m57.267"],
+        "MQQ":    ["N31d49m52.344", "W81d30m36.648", "Mc Intosh"],
+        "NOMAS":  ["N32d20m54.130", "W80d52m35.932"],
+        "PLZZZ":  ["N32d43m50.003", "W81d13m05.983"],
+        "REBUS":  ["N32d27m44.115", "W81d00m12.329"],
+        "RICKY":  ["N32d04m15.777", "W81d05m38.129"],
+        "SAV":    ["N32d08m46.563", "W81d11m56.773", "Savannah"],
+        "SINBY":  ["N32d07m38.890", "W81d05m06.814"],
+        "SMALZ":  ["N32d28m33.003", "W81d31m40.991"],
+        "SOOOP":  ["N32d22m22.996", "W80d33m50.983"],
+        "SOVIE":  ["N32d07m47.138", "W81d20m09.427"],
+        "SPONG":  ["N32d29m36.132", "W81d23m13.036"],
+        "SUTME":  ["N32d20m03.799", "W81d12m05.944"],
+        "SVN":    ["N32d00m42.999", "W81d08m27.095", "Hunter"],
+        "TELOE":  ["N32d07m34.354", "W80d58m02.690"],
+        "TERBE":  ["N31d54m44.803", "W81d11m54.410"],
+        "TILEE":  ["N32d07m50.393", "W81d27m14.074"],
+        "TILLS":  ["N32d28m44.099", "W81d14m04.844"],
+        "TOSEA":  ["N32d14m03.037", "W81d12m03.199"],
+        "VIYAP":  ["N31d15m08.147", "W81d26m08.188"],
+        "WADEB":  ["N32d07m55.529", "W81d41m22.358"],
+        "WISPR":  ["N32d01m58.857", "W81d13m30.705"],
+        "WIUMS":  ["N32d10m46.767", "W80d46m05.091"],
+        "WOHPY":  ["N31d50m16.922", "W81d12m37.201"],
+        "WORIB":  ["N32d02m33.756", "W80d58m07.305"],
+        "WOSUG":  ["N31d54m42.722", "W81d17m46.852"],
+        "WOXIT":  ["N32d12m51.009", "W81d27m11.107"],
+        "YUHVY":  ["N32d11m41.727", "W81d07m18.269"],
+        "ZALAV":  ["N32d08m24.287", "W80d49m24.824"],
+        "ZUTAN":  ["N31d54m46.609", "W81d06m01.969"]
+    },
+    "restricted": [
+        {
+            "name": "R-3005A",
+            "height": "29000ft",
+            "poly": [
```

Comment:
Polys should be defined as a series of coordinates, not a series of lines.

Additionally, it seems some of the coordinates aren't quite right. Ideally, the restricted area should line up with the video map perfectly

![image](https://user-images.githubusercontent.com/13125311/223106317-554b52a7-2fe1-437f-ba04-007c0dbc5148.png)

![image](https://user-images.githubusercontent.com/13125311/223106332-f631e21b-e399-415d-8888-b746bc64b694.png)